core/controllers/admin.py

```diff
@@ -633,3 +633,64 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class AddReviewerHandler(base.BaseHandler):
+    """Handles adding reviewer for community dashboard page."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        new_reviewer_username = self.payload.get('username')
+        new_reviewer_user_id = (
+            user_services.get_user_id_from_username(new_reviewer_username))
+
+        if new_reviewer_user_id is None:
+            raise self.InvalidInputException(
+                'Invalid username: %s' % new_reviewer_username)
+
+        user_allowed_to_review = self.payload.get('review')
```

Comment:
This variable name seems odd -- it's not a user. Could you rename it to illustrate more clearly what it is actually naming?